Fix: Add global.conf description section (#3561)
conan-io/docs: reference/config_files/global_conf.rst

.. include:: ../../common/config_list.inc

Description of configurations
+++++++++++++++++++++++++++++

core.cache:storage_path
^^^^^^^^^^^^^^^^^^^^^^^

Absolute path to a folder where the Conan packages and the database of the packages will be stored.

> This folder will take the most of the space in your disk, as it stores the binary pacakges dowloaded or created.

Reviewer: ```suggestion
This folder will take most of the space in your disk if left unchecked, as it stores both the binary packages dowloaded or created.
```

Author: what do you mean by that "if left unchecked"?

Author: just to clarify, I wanted to say that this folder will be the heaviest conan storage folder. Not that this will take the space of the user's disk for sure 😄

> This folder will take the most of the space in your disk, as it stores the binary pacakges dowloaded or created.

Reviewer: ```suggestion
This folder will be the heaviest Conan storage folder, as it stores the binary packages dowloaded or created.
```

.. code-block:: text
    :caption: *global.conf*

    core.cache.storage_path = C:\Users\danielm\my_conan_storage_folder

**Default value:** ``<CONAN_HOME>/p``

core.download:download_cache
^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> Absolute path to a folder where the Conan packages *uncompressed* will be stored.

Reviewer: ```suggestion
Absolute path to a folder where the Conan packages will be stored *uncompressed*.
```

But are you sure about the uncompressed? I though this were for the `conan_package.tar.gz` files and such

Author: sorry, I meant just the opposite 🤣 
```suggestion
Absolute path to a folder where the Conan packages will be stored *compressed*.
```

Reviewer: :P

.. _reference_config_files_global_conf_ssl_certificates:

Configuration of client certificates

> ++++++++++++++++++++++++++++++++++++

Author: Had to change this to be consistent with header fromatting (it was an error otherwise)
